examples/features/nse-composition/nse-composition-ns.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: networkservicemesh.io/v1
+kind: NetworkService
+metadata:
+  name: nse-composition
```

Comment:
In README file you define `PASS_COUNT` variable, but here in network service definition you hardcode the number of passthrough endpoints.
It's counter-intuitive that you can't the value of `PASS_COUNT`.

I think it would be better to generate ns definition file too.

Reply:
done